schemas/test/en/test_placeholder_first_non_empty_item.json

```diff
@@ -0,0 +1,283 @@
+                                                        "date_format": "d MMMM yyyy"
+                                                    }
+                                                }
+                                            ]
+                                        }
+                                    ]
+                                },
+                                "type": "General",
+                                "answers": [
+                                    {
+                                        "id": "date-answer",
+                                        "mandatory": true,
+                                        "type": "Radio",
+                                        "options": [
+                                            {
+                                                "label": "Yes, I can report for this period",
+                                                "value": "Yes, I can report for this period"
+                                            },
+                                            {
+                                                "label": "No, I need to report for a different period",
+                                                "value": "No, I need to report for a different period"
+                                            }
+                                        ]
+                                    }
+                                ]
+                            },
+                            "routing_rules": [
+                                {
+                                    "block": "total-turnover-block",
+                                    "when": {
+                                        "in": [
+                                            {
+                                                "identifier": "date-answer",
+                                                "source": "answers"
+                                            },
+                                            ["Yes, I can report for this period"]
+                                        ]
+                                    }
+                                },
+                                {
+                                    "block": "date-entry-block"
+                                }
+                            ]
+                        },
+                        {
+                            "id": "date-entry-block",
+                            "type": "Question",
+                            "question": {
+                                "id": "date-entry-question",
+                                "title": "What are the dates of the period that you will be reporting for?",
+                                "guidance": {
+                                    "contents": [
+                                        {
+                                            "description": "Enter a date between 24 June 2023 and 2 August 2023"
```

Comment:
This doesn't match with the allowed values.

Reply:
Do they change this every month now? Changed it earlier this month too. Updated 🖖

Reply:
The launcher dates changed sometime recently to use current date, we should change back.